packages/bot-web-ui/.eslintrc.js

```diff
@@ -17,51 +17,4 @@ module.exports = {
             webpack: { config: webpackConfig({}) },
         },
     },
-    plugins: ['simple-import-sort'],
-    rules: {
-        'simple-import-sort/imports': 'warn',
-        'simple-import-sort/exports': 'warn',
-    },
-    overrides: [
-        {
-            files: ['**/*.js', '**/*.ts', '**/*.tsx', '**/*.jsx'],
-            rules: {
-                'simple-import-sort/imports': [
-                    'warn',
-                    {
-                        groups: [
-                            [
-                                'public-path',
-                                // `react` first, then packages starting with a character
-                                '^react$',
-                                '^[a-z]',
-                                // Packages starting with `@`
-                                '^@',
-                                // Packages starting with `~`
-                                '^~',
-                                '^Components',
-                                '^Constants',
-                                '^Utils',
-                                '^Types',
-                                '^Stores',
-                                // Imports starting with `../`
-                                '^\\.\\.(?!/?$)',
-                                '^\\.\\./?$',
-                                // Imports starting with `./`
-                                '^\\./(?=.*/)(?!/?$)',
-                                '^\\.(?!/?$)',
-                                '^\\./?$',
-                                // Style imports
-                                '^.+\\.s?css$',
-                                // Side effect imports
-                                '^\\u0000',
-                                // Delete the empty line copied as the next line of the last import
-                                '\\s*',
```

Comment:
I think it would be even better if this rule is in the global config because if a file has 30 lines of imports and empty lines between them, the file will be less readable.

Reply:
<img width="619" alt="Screenshot 2023-09-13 at 11 46 03" src="https://github.com/binary-com/deriv-app/assets/103181650/bc4954e8-6524-4569-91d5-6930257620d9">

Reply:
@maryia-matskevich-deriv I believe empty lines between imports will add categorisation to the imports and makes the code more readable. 
@ali-hosseini-deriv was there any concern when you asked to removing this empty line in the first place?

Reply:
not a specific concern for that. 
i'm ok with that.